plugin-script/src/main/java/io/kestra/plugin/scripts/exec/scripts/models/DockerOptions.java

```diff
@@ -0,0 +1,210 @@
+package io.kestra.plugin.scripts.exec.scripts.models;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.micronaut.core.annotation.Introspected;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+import java.util.Map;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+@SuperBuilder
+@NoArgsConstructor
+@Getter
+@Introspected
+public class DockerOptions {
+    @Schema(
+        title = "Docker api uri"
+    )
+    @PluginProperty(dynamic = true)
+    private String host;
+
+    @Schema(
+        title = "Docker config file",
+        description = "Full file that can be used to configure private registries, ..."
+    )
+    @PluginProperty(dynamic = true)
+    private String config;
+
+    @Schema(
+        title = "Docker image to use"
+    )
+    @PluginProperty(dynamic = true)
+    @NotNull
+    @NotEmpty
+    protected String image;
+
+    @Schema(
+        title = "Docker user to use"
+    )
+    @PluginProperty(dynamic = true)
+    protected String user;
+
+    @Schema(
+        title = "Docker entrypoint to use"
+    )
+    @PluginProperty(dynamic = true)
+    protected List<String> entryPoint;
+
+    @Schema(
+        title = "Docker extra host to use"
+    )
+    @PluginProperty(dynamic = true)
+    protected List<String> extraHosts;
+
+    @Schema(
+        title = "Docker network mode to use"
+    )
+    @PluginProperty(dynamic = true)
+    protected String networkMode;
+
+    @Schema(
+        title = "List of volumes to mount",
+        description = "Must be a valid mount expression as string, example : `/home/user:/app`\n\n" +
+            "Volumes mount are disabled by default for security reasons, you must enabled on server configuration with `kestra.tasks.scripts.docker.volume-enabled` to `true`"
+    )
+    @PluginProperty(dynamic = true)
+    protected List<String> volumes;
+
+    @Schema(
+        title = "The pull policy for an image",
+        description = "Can be used to prevent pulling of image locally build and to avoid pull again the same image"
```

Comment:
```suggestion
        description = "Pull policy can be used to prevent pulling of an already existing image `IF_NOT_PRESENT`, or can be set to `ALWAYS` to pull the latest version of the image even if an image with the same tag already exists."
```